packages/bpk-tokens/src/ios/base/borders.json

```diff
@@ -35,6 +47,26 @@
     "CORNER_RADIUS_PILL": {
       "value": "{!BORDER_RADIUS_PILL}",
       "category": "radii"
+    },
+    "BORDER_RADIUS_XS": {
```

Comment:
Do we need these given that they are named corner radius above? It seems we are just creating duplicates of the same tokens 🤔

Reply:
See https://github.com/Skyscanner/backpack/pull/2006#discussion_r509957948